spec/lib/importers/licence_transaction/licence_importer_spec.rb

```diff
@@ -116,7 +116,7 @@ def expected_put_content_payload
       change_note: "Imported from Publisher",
       schema_name: "specialist_document",
       publishing_app: "specialist-publisher",
-      rendering_app: "government-frontend",
+      rendering_app: "frontend",
```

Comment:
I think all the changes in this commit should be fixed up to the original commit.